app.R

```diff
@@ -72,29 +97,31 @@ ui <- fluidPage(
               tabPanel("1 Single",
                       sidebarLayout(position = "left",
                                     sidebarPanel(
-                                      actionButton("resetSingleForm", "Clear form!"),
-                                      actionButton("randomSpecies", "Random species!"),
+                                      #actionButton("resetSingleForm", "Clear form!"),
+                                      #actionButton("randomSpecies", "Random species!"),
   
                                       fluidRow(
                                         column(8, align="center", offset = 2,
-                                               tags$h4("Enter species")
+                                               tags$h4("Enter a species"),
+                                                tags$h4("or click '?' for a random choice:")
                                                #actionButton("randomSpecies", "Random species!")
                                         )
                                       ),
                                       
-                                      #fluidRow(
-                                      #  column(4, 
-                                      #         textInput("speciesinput", label = "Enter species",
-                                      #                   placeholder = "Aloe zebrina")
-                                      #         ),
-                                      #  column(4, offset = 2,
-                                      #         actionButton("randomSpecies", "Random")
-                                      #  )
-                                      #),
+                                      fluidRow(
+                                        column(10, 
+                                               style='font-size: 10px; margin-top:-1.4em',
```

Comment:
To avoid needing this styling, you can just set `label = NULL` inside the text input definition to make the button and the input sit on the same line.

I would also change the button to say random. You can remove any padding on the button to make it sit closer to the text input, like this:
```
fluidRow(
  column(8, textInput("speciesinput", label=NULL, placeholder="Aloe zebrina")),
  column(2, actionButton("randomSpecies", "Random"), offset=0, style="padding:0px;")
),
```

Reply:
yes, have tidied this up now and label=NULL fixes it